### DISCUSSION TITLE
RTK-Query's pollingInterval is firing after component unmounts

### DISCUSSION THREAD

**Discussion by dorbaruchh** [reduxjs/redux-toolkit#1655] [Q&A]
Hey guys,
I'm using a RTK-Query hook and passing in pollingInterval to make the request trigger every minute. 
I noticed that after I log out and the component where the hook is used completely unmounts I can still see one more request sent after the interval passes. After this last request is sent no more requests are sent afterward. 
Is this expected behavior? Is there any way to prevent that last request from firing?

**Comment by phryneas** [comment]
That sounds like a bug. Could you create a small reproduction sandbox and open an issue?
